### ISSUE TITLE
`Error: $ operator is invalid for atomic vectors` when installing a GitHub package

### ISSUE THREAD

**Issue by eliotmcintire** [PredictiveEcology/Require#33]
```
> Require("PredictiveEcology/LandR@cluster2")
Installing: LandR
Error: $ operator is invalid for atomic vectors
In addition: Warning message:
In FUN(X[[i]], ...) :
  DESCRIPTION file of package 'LandR' is missing or broken
```
with `traceback()`:
```
> traceback()
11: FUN(X[[i]], ...)
10: lapply(X = X, FUN = FUN, ...)
9: sapply(pkgDesc, function(x) !is.null(x$Priority) && x$Priority == 
       "base")
8: sessionInfo()
7: doInstalls(pkgDT, install_githubArgs = install_githubArgs, install.packagesArgs = install.packagesArgs, 
       install = install, ...)
6: Require::Require(out[needAction]$packageFullName, require = FALSE, 
       upgrade = FALSE)
```

**Comment by achubaty** [comment]
@eliotmcintire i assume this is now resolved in the development branch?

**Comment by achubaty** [comment]
resolved in v0.1.4
